irobot_real_move:
  ros__parameters:
    print_data: True
    samplingtime: 0.1
    dis_change_goal: 1.0
    goal_tolerance: 3.0
    T: 5.0
    robot_threshold: 0.35   
    dis_safe_obstacle: 0.45
    num_vel_rollouts: 4
    num_ang_vel_rollouts: 31
    weight_angle_row: 0.02
    weight_angle_turn: 0.5
    weight_angle_goal: 0.3
    weight_vel: 0.1
    weight_obs: 0.1
    weight_spr_row: 0.9
    weight_spr_turn: 0.0
    weight_spr_goal: 0.05
    score_obstacle: 2.0
    max_acc: 3.0
    max_ang_acc: 5.0
    lim_max_vel: 0.4
    lim_min_vel: 0.1
    lim_max_ang_vel: 0.78539

    topic_global_frame: '/mavros/local_position/pose'           # gps_indoor   odom     /mavros/local_position/pose 
    topic_imu: '/mavros/local_position/pose'          # /zedx/right/imu/data  /mavros/local_position/pose 
    base_link_frame: 'base_link'

irobot_gazebo_move:
  ros__parameters:
    T: 5.0
    samplingtime: 0.1
    num_vel_rollouts: 4
    num_ang_vel_rollouts: 41
    robot_threshold: 0.35 
    dis_safe_obstacle: 0.45
    map_resolution: 0.25
    map_origin_x: -5.0
    map_origin_y: -5.0
    weight_angle_goal: 0.6
    weight_vel: 0.1
    weight_obs: 1.0
    score_obstacle: 2.0
    max_acc: 3.0
    max_ang_acc: 3.0
    lim_max_vel: 1.0
    lim_min_vel: 0.0
    lim_max_ang_vel: 1.5732
    goal_shift: 0.75
